docs/4.1/enterprise/ssh_fips.md

```diff
@@ -0,0 +1,150 @@
+| [AU-03 Audit and Accountability](https://nvd.nist.gov/800-53/Rev4/control/AU-3) – Content of Audit Records and [AU-12 Audit Generation](https://nvd.nist.gov/800-53/Rev4/control/AU-12) | Teleport contains an [Audit Log](../architecture/teleport_auth/#audit-log) that records cluster-wide events such as: <br>• Failed login attempts.<br>• Commands that were executed (SSH “exec” commands).<br> • Ports that were forwarded. <br>• File transfers that were initiated.|
+| [AU-10 Non-Repudiation](https://nvd.nist.gov/800-53/Rev4/control/AU-10)  | Teleport audit logging supports both events as well as audit of an entire SSH session. For non-repudiation purposes a full session can be replayed back and viewed.  |
+| [CM-08 Information System Component Inventory](https://nvd.nist.gov/800-53/Rev4/control/CM-8)  | Teleport maintains a live list of all nodes within a cluster. This node list can be queried by users (who see a subset they have access to) and administrators anytime.|
+| [IA-03 Device Identification and Authentication](https://nvd.nist.gov/800-53/Rev4/control/IA-3)  | Teleport requires valid x509 or SSH certificates issued by a Teleport Certificate Authority (CA) to establish a network connection for device-to-device network connection between Teleport components. |
+| [SC-12 Cryptographic Key Establish and Management](https://nvd.nist.gov/800-53/Rev4/control/SC-12)  | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed). <br><br>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
+
+
+
+
+
+Enterprise customers can download the custom FIPS package from the [Gravitational Dashboard](https://dashboard.gravitational.com/web/). 
+Look for `Linux 64-bit (FedRAMP/FIPS)`, with RPM and DEB packages easily available. 
+
+# Setup
+Customer can follow our [Enterprise Quickstart](quickstart-enterprise.md) for basic 
+instructions on how to setup Teleport Enterprise. You'll need to start with the Teleport 
+Enterprise FIPS Binary. 
+
+After downloading the binary tarball, run:
+
+```bsh
+$ tar -xzf teleport-ent-v4.1.4-linux-amd64-fips-bin.tar.gz
+$ cd teleport-ent
+$ sudo ./install 
+# This will copy Teleport Enterprise to /usr/local/bin.
+```
+
+## Configuration 
+
+### Teleport Auth Server
+Now, save the following configuration file as `/etc/teleport.yaml` on the _auth.example.com_:
```

Comment:
either "on the auth server" or "on _auth.example.com" (no the with a domain name)